source/openapi-admin-v3.yaml

```diff
@@ -847,48 +847,70 @@ paths:
               schema:
                 items:
                   anyOf:
-                    - $ref: "#/components/schemas/AtlasCluster"
+                    - $ref: "#/components/schemas/ServiceResponse"
```

Comment:
I think we should remove the `anyOf` here and only use `ServiceResponse`

<img width="259" alt="Screenshot 2023-10-12 at 11 39 38 AM" src="https://github.com/mongodb/docs-app-services/assets/15657698/59167688-0f73-4bbf-8a8d-cbbe2647493e">

Reply:
I'd love to know why.

Reply:
> I'd love to know why.

Per that screenshot, the endpoint only returns `ServiceResponse` objects